### ISSUE TITLE
mobile theme

### ISSUE THREAD

**Issue by hermidalc** [CBIIT/nci-ctd2-dashboard#163]
*Originally reported by*: **mcmanus-RNA**

Would be nice to have a mobile theme- even if it doesn't retain all the functionalities, text content could be more appropriately displayed.

**Comment by hermidalc** [comment]
*Original comment by*: @armish

a few stling hacks to make the site more mobile friendly --

**Comment by hermidalc** [comment]
*Original comment by*: @armish

Hi Michael,

Thanks for reporting this. I tried to make sure the website works with the mobile browsers, but looks like with the latest changes, I broke a few things on the mobile version. For the records, I attached two images that show problematic rendering on mobile. Will fix this as soon as possible.

![Screenshot_2014-07-29-13-51-39.png](https://bitbucket.org/repo/ayGMep/images/3968502928-Screenshot_2014-07-29-13-51-39.png)
![Screenshot_2014-07-29-13-52-03.png](https://bitbucket.org/repo/ayGMep/images/631759938-Screenshot_2014-07-29-13-52-03.png)
